dali/test/python/test_dltensor_operator.py

```diff
@@ -153,13 +182,152 @@ def mxnet_cast(in1, in2):
     return [mxnd.cast(t, dtype='float32') for t in in1], [mxnd.cast(t, dtype='int64') for t in in2]
```

Comment:
Do we need/can do any sync here for MXNet?

Reply:
It's implicitly synchronized when converting output tensors to DLPack tensors